pom.xml

```diff
@@ -234,6 +245,12 @@
             <scope>test</scope>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
```

Comment:
why is this now needed for tests?

Reply:
I didn't have the time until now to look into this again, it's in the pipe, but no idea when/if I will **really** be able to get to it again. Hopefully yes :)

Reply:
That's to resolve a Maven enforcer error.